packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/index.tsx

```diff
@@ -155,69 +156,62 @@ const SliceZone: React.FC<SliceZoneProps> = ({
   };
 
   return (
-    <Box mt={3}>
-      <ZoneHeader
-        Heading={
-          <div style={{ display: "flex", alignItems: "center" }}>
-            <Heading as="h6" style={{ marginRight: vars.size[8] }}>
-              Slice Zone
-            </Heading>
-            <Switch
-              checked={!!sliceZone}
-              onCheckedChange={(checked) => {
-                if (checked) {
-                  onCreateSliceZone();
-                } else {
-                  setIsDeleteSliceZoneModalOpen(true);
-                }
-              }}
-            />
-          </div>
-        }
-        Actions={
-          <Flex sx={{ alignItems: "center" }}>
-            {sliceZone ? (
-              <Text pr={3} sx={{ fontSize: "14px" }}>
-                data.{sliceZone.key}
-              </Text>
-            ) : null}
-            <Flex sx={{ gap: "8px" }}>
-              <Button
-                variant="secondary"
-                startIcon={<Icon name="add" />}
-                onClick={onCreateNewSlice}
-              >
+    <Box flexDirection="column">
+      <List>
+        <ListHeader
+          actions={
+            <>
+              <Button onClick={onCreateNewSlice} startIcon="add">
                 New slice
               </Button>
-              {availableSlices.length > 0 && (
+              {availableSlices.length > 0 ? (
                 <Button
-                  variant="secondary"
-                  startIcon={<Icon name="edit" />}
-                  onClick={onAddNewSlice}
                   data-cy="update-slices"
+                  onClick={onAddNewSlice}
+                  startIcon="edit"
                 >
                   Update Slices
                 </Button>
-              )}
-            </Flex>
-          </Flex>
-        }
-      />
-      {sliceZone && !slicesInSliceZone.length ? (
-        <SliceZoneBlankSlate
-          onAddNewSlice={onAddNewSlice}
-          onCreateNewSlice={onCreateNewSlice}
-          projectHasAvailableSlices={availableSlices.length > 0}
-        />
-      ) : (
-        <SlicesList
-          slices={slicesInSliceZone}
-          modelsStatuses={modelsStatuses}
-          authStatus={authStatus}
-          isOnline={isOnline}
-          format={customType.format}
-        />
-      )}
+              ) : undefined}
+            </>
+          }
+          toggle={
+            customType.format !== "page" || customType.tabs[0].key !== tabId ? (
```

Comment:
❓You changed the condition here, right? Before it was always possible to remove the slice zone, even the first tab. I think what you did is good but prefer to ask